sentiment_analysis/app/app.py

```diff
@@ -66,11 +66,6 @@ def cape_handler(text):
         result = "positive"
     else:
         result = "negative"
-    return (
-        "The sentiment is: ",
-        result,
-        " with a probability of ",
-        output_data[0][output_result] * 100,
-        "%.",
-    )
-    
\ No newline at end of file
+    
+    prob = output_data[0][output_result] * 100
+    return (str(float(f'{prob:.2f}')) + "% " + result) or "You've stumped me! Please try a different phrase."
```

Comment:
When would this `or` happen? Feels a little hard to parse what this is doing for a reference function 🤔

Reply:
☝️ agree with ben

also, for the string output, can you do `f'{prob:.2f}% {result}'` ?

Reply:
I don't think that the ```or``` is needed. Words that are not in the model's vocabulary are handled to be skipped, so that if there is an unknown word in a sentence, the model will simply assess all known words.

Reply:
Nevertheless, the ```or``` does not hurt. I cannot think of a situation when it will be executed, but better to have it just in case. Regarding the formatting, it looks good to me. I tested it and it renders nicely as follows:

78.08% positive

Hence, I approve the PR.

Reply:
I defer to @ekloberdanz's approval and will leave the `or` since it's her function :)